src/App.js

```diff
@@ -1,11 +1,19 @@
 import React from "react";
+import { Route } from "react-router-dom";
+import Home from "./Home";
+import PizzaForm from "./PizzaForm";
+import "./App.css";
 
 const App = () => {
   return (
-    <>
+    <div className="container">
       <h1>Lambda Eats</h1>
-      <p>You can remove this code and create your own header</p>
-    </>
+      <p>Code and eat, eat and code.</p>
+
+      <Route exact path='/' component={Home} />
```

Comment:
there is a newer syntax that allows easier use of props when using react-router
`<Route path='/'><Home /></Route>`

I've always found it's a little less clunky than the component= syntax.